### DISCUSSION TITLE
Funky discussions are now open

### DISCUSSION THREAD

**Discussion by pukmajster** [pukmajster/funky#19] [Announcements]
## Welcome!

This is the perfect place to post your questions and ideas regarding Funky. Should you have a bug report, please resort to the Issues page. Remember to be civil!

**Comment by Monolith0000** [comment]
Hi! This app is gorgeous, good work on it! I just had a idea that I thought might be worth raising. In MO2 for example, mods are listed and can be placed in user created categories, for example (for L4D2), this could be "Characters", "Weapons", "VFX" and things like that, for the sake of organisation. I think that would take this app to a whole new level! Would it be possible for these features to be implemented? Thank you for your hard work!
